### ISSUE TITLE
airmon-ng does not find wifi0

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#163]
**Reported by anonymous on 10 Apr 2007 19:46 UTC**

When starting airmon-ng no wifi0 is displayed:

```
blacknote:/# iwconfig
lo        no wireless extensions.
eth0      no wireless extensions.
irda0     no wireless extensions.
wifi0     no wireless extensions.
ath0      IEEE 802.11g  ESSID:"WLAN"
          Mode:Managed  Frequency:2.437 GHz  Access Point: 00:04:0E:68:9C:B4
          Bit Rate:48 Mb/s   Tx-Power:17 dBm   Sensitivity=0/3
          Retry:off   RTS thr:off   Fragment thr:off
          Encryption key:3338-3937-3431-3633-3936-3739-35   Security mode:restricted
          Power Management:off
          Link Quality=27/94  Signal level=-71 dBm  Noise level=-98 dBm
          Rx invalid nwid:770  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0
sit0      no wireless extensions.
blacknote:/# airmon-ng
usage: airmon-ng <start|stop> <interface> [channel]

Interface       Chipset         Driver
ath0            Atheros         madwifi-ng VAP (parent: wifi0)
blacknote:/# 
```

Also executing "airmon-ng start wifi0 6" does not work :/
I've installed the drivers,including the patch provided on aircrack-ng.org ,from sources and
it seems they are patched correctly(I get the interface-name bug when setting the monitor mode via wlanconfig).

But I can't user airplay to do a fake-auth when setting the monitor mode per hand.
It tells me the network is down :(

A lot of Thanks

Oh and sorry for the terrible englisch. (don't know if its that bad,you could tell me =))

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 10 Apr 2007 19:46 UTC**

airmon-ng never worked for me. Destroying a VAP is no problem, but creating one in monitor mode seems impossible via airmon-ng.

**Comment by aircrack-ng** [comment]
**Modified by misterx on 10 Apr 2007 19:46 UTC**

**Comment by aircrack-ng** [comment]
**Modified by misterx on 10 Apr 2007 19:46 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 10 Apr 2007 19:46 UTC**

Could you run the following as root and report the output

```
find /proc 2>/dev/null | grep fftx
```

**Comment by aircrack-ng** [comment]
**Comment by obo on 10 Apr 2007 19:46 UTC**

wifi0     no wireless extensions.
Doesn't this seems to indicate that the problem isn't with airmon? Are you sure you're using the correct driver?

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 10 Apr 2007 19:46 UTC**

Not at all, that's normal, wifi0 is the madwifi control interface. Airmon-ng works fine for me, it seems not to work for everybody.

**Comment by aircrack-ng** [comment]
**Modified by misterx on 10 Apr 2007 19:46 UTC**

**Comment by aircrack-ng** [comment]
**Comment by Kunipert on 10 Apr 2007 19:46 UTC**

Hi,
I have the same issue (Read here: http: //tinyshell.be/aircrackng/forum/index.php?topic=1893.0 )
```root@hornisse:~# find /proc 2>/dev/null | grep fftx
/proc/sys/dev/wifi0/fftxqmin```

**Comment by aircrack-ng** [comment]
**Comment by misterx on 10 Apr 2007 19:46 UTC**

It's strange because it's exactly where airmon-ng is looking.
So, there's maybe something wrong with the command to search the interface.

Could you run 'ifconfig -a' and post the result here.
NB: put the 3 { on a line then append a new line then paste the result, append a new line then put the 3 closing braces '}' (and use preview if you're not sure it will be good).

**Comment by aircrack-ng** [comment]
**Comment by Kunipert on 10 Apr 2007 19:46 UTC**

Output from ifconfig -a and iwconfig directly after boot:
```
root@hornisse:~# ifconfig -a
ath0      Protokoll:Ethernet  Hardware Adresse 00:20:A6:52:1E:BB  
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          Kollisionen:0 Sendewarteschlangenlnge:0 
          RX bytes:0 (0.0 b)  TX bytes:0 (0.0 b)

eth0      Protokoll:Ethernet  Hardware Adresse 00:0D:56:36:7C:11  
          inet Adresse:192.168.0.43  Bcast:192.168.0.255  Maske:255.255.255.0
          inet6 Adresse: fe80::20d:56ff:fe36:7c11/64 Gltigkeitsbereich:Verbindung
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:65 errors:0 dropped:0 overruns:0 frame:0
          TX packets:12 errors:0 dropped:0 overruns:0 carrier:0
          Kollisionen:0 Sendewarteschlangenlnge:1000 
          RX bytes:4918 (4.8 KiB)  TX bytes:1010 (1010.0 b)
          Interrupt:7 

lo        Protokoll:Lokale Schleife  
          inet Adresse:127.0.0.1  Maske:255.0.0.0
          inet6 Adresse: ::1/128 Gltigkeitsbereich:Maschine
          UP LOOPBACK RUNNING  MTU:16436  Metric:1
          RX packets:3 errors:0 dropped:0 overruns:0 frame:0
          TX packets:3 errors:0 dropped:0 overruns:0 carrier:0
          Kollisionen:0 Sendewarteschlangenlnge:0 
          RX bytes:188 (188.0 b)  TX bytes:188 (188.0 b)

wifi0     Protokoll:UNSPEC  Hardware Adresse 00-20-A6-52-1E-BB-00-00-00-00-00-00-00-00-00-00  
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:663 errors:0 dropped:0 overruns:0 frame:12
          TX packets:72 errors:0 dropped:0 overruns:0 carrier:0
          Kollisionen:0 Sendewarteschlangenlnge:199 
          RX bytes:80126 (78.2 KiB)  TX bytes:3312 (3.2 KiB)
          Interrupt:11 
```

```
root@hornisse:~# iwconfig
lo        no wireless extensions.

eth0      no wireless extensions.

wifi0     no wireless extensions.

ath0      IEEE 802.11g  ESSID:""  Nickname:""
          Mode:Managed  Frequency:2.437 GHz  Access Point: Not-Associated   
          Bit Rate:0 kb/s   Tx-Power:0 dBm   Sensitivity=0/3  
          Retry:off   RTS thr:off   Fragment thr:off
          Encryption key:off
          Power Management:off
          Link Quality=0/70  Signal level=-89 dBm  Noise level=-89 dBm
          Rx invalid nwid:1860  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0
```

**Comment by aircrack-ng** [comment]
**Comment by misterx on 10 Apr 2007 19:46 UTC**

(In #458) Fixed madwifi-ng wifiX detection. It is due to translation in ifconfig (Closes: #163)
